### PR TITLE
fix json examples for --openshift-delegate-urls

### PR DESCRIPTION
**Pull request by dastrobu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,14 +116,14 @@ Cons:
...
+    --openshift-delegate-urls='{"/":{"group":"custom.group","resource":"myproxy","verb":"get"}}'
 
     # Grant access only to paths under /api
-    --openshift-delegate-urls='{"/api":{"group":"custom.group","resource":"myproxy""verb":"get"}}'
+    --openshift-delegate-urls='{"/api":{"group":"custom.group","resource":"myproxy","verb":"get"}}'
 
 WARNING: Because users are sending their own credentials to the proxy, it's important to use this 
 setting only when the proxy is under control of the cluster administrators. Otherwise, end users 
-may be unwittingly provide their credentials to untrusted components that can then act as them.
+may be unwittingly to provide their credentials to untrusted components that can then act as them.
```
Comment (VassilisVassiliadis) [inline README.md:126]:
```suggestion
may unwittingly provide their credentials to untrusted components that can then act as them.
```

Comment (dastrobu) [reply README.md]: Thanks for the remark. It's fixed.

**Comment by stlaz** [comment]
/approve
/remove-lifecycle rotten

thanks for the PR and sorry about the delay :neutral_face:

**Pull request merged**